### PR TITLE
revise cleanData to cleanPolicy

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? (Required)
revise cleanData to cleanPolicy

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?
https://github.com/pingcap/tidb-operator/pull/3002

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by lichunzhu** [comment]
@DanielZhangQD @ran-huang PTAL

File: zh/backup-to-aws-s3-using-br.md [hunk 1]
```diff
@@ -331,7 +331,14 @@ kubectl get bk -n test1 -o wide
 
     如果发现 `tikv_gc_lifetime` 值过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
 
-* `.spec.cleanData`：设置为 true 时删除该 Backup CR 时会同时清除该 CR 备份出的数据，默认为 false。值得注意的是，在 v1.1.2 以及之前版本不存在该字段，且默认在删除 CR 的同时删除备份的文件。若 v1.1.3 及之后版本的用户希望保持该行为，需要设置该字段为 true。
+* `.spec.cleanPolicy`：备份集群后删除备份 CR 时的备份文件清理策略。目前支持三种清理策略：
+
+    `Retain`：任何情况下，删除备份 CR 时会保留备份出的文件
+    `Delete`：任何情况下，删除备份 CR 时会删除备份出的文件
+    `OnFailure`：仅在备份中失败时，删除备份 CR 时会删除备份出的文件
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:338]:
```suggestion
    `OnFailure`：如果备份失败，删除备份 CR 时会删除备份出的文件
```

File: zh/backup-to-aws-s3-using-br.md [hunk 2]
```diff
@@ -331,7 +331,14 @@ kubectl get bk -n test1 -o wide
...
+    
+    默认时不填写该字段与填写上述字段以外的字段均会保留备份出的文件。值得注意的是，在 v1.1.2 以及之前版本不存在该字段，且默认在删除 CR 的同时删除备份的文件。若 v1.1.3 及之后版本的用户希望保持该行为，需要设置该字段为 `Delete`。
```
Comment (DanielZhangQD) [inline zh/backup-to-aws-s3-using-br.md:340]:
```suggestion
    如果不配置该字段或者配置该字段的值为上述三种以外的值均会保留备份出的文件。值得注意的是，在 v1.1.2 以及之前版本不存在该字段，且默认在删除 CR 的同时删除备份的文件。若 v1.1.3 及之后版本的用户希望保持该行为，需要设置该字段为 `Delete`。
```

File: zh/backup-to-aws-s3-using-br.md [hunk 3]
```diff
@@ -331,7 +331,14 @@ kubectl get bk -n test1 -o wide
 
     如果发现 `tikv_gc_lifetime` 值过大（通常为 10m），则需要按照[调节 `tikv_gc_lifetime`](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#导出大规模数据时的-tidb-gc-设置) 将 `tikv_gc_lifetime` 调回原样。
 
-* `.spec.cleanData`：设置为 true 时删除该 Backup CR 时会同时清除该 CR 备份出的数据，默认为 false。值得注意的是，在 v1.1.2 以及之前版本不存在该字段，且默认在删除 CR 的同时删除备份的文件。若 v1.1.3 及之后版本的用户希望保持该行为，需要设置该字段为 true。
+* `.spec.cleanPolicy`：备份集群后删除备份 CR 时的备份文件清理策略。目前支持三种清理策略：
+
+    `Retain`：任何情况下，删除备份 CR 时会保留备份出的文件
+    `Delete`：任何情况下，删除备份 CR 时会删除备份出的文件
+    `OnFailure`：如果备份中失败，删除备份 CR 时会删除备份出的文件
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:338]:
```suggestion
    * `Retain`：任何情况下，删除备份 CR 时会保留备份出的文件
    * `Delete`：任何情况下，删除备份 CR 时会删除备份出的文件
    * `OnFailure`：如果备份中失败，删除备份 CR 时会删除备份出的文件
```

File: zh/backup-to-aws-s3-using-br.md [hunk 4]
```diff
@@ -331,7 +331,14 @@ kubectl get bk -n test1 -o wide
...
+    
+    如果不配置该字段或者配置该字段的值为上述三种以外的值均会保留备份出的文件。值得注意的是，在 v1.1.2 以及之前版本不存在该字段，且默认在删除 CR 的同时删除备份的文件。若 v1.1.3 及之后版本的用户希望保持该行为，需要设置该字段为 `Delete`。
```
Comment (ran-huang) [inline zh/backup-to-aws-s3-using-br.md:340]:
```suggestion
    如果不配置该字段，或者配置该字段的值为上述三种以外的值，均会保留备份出的文件。值得注意的是，在 v1.1.2 以及之前版本不存在该字段，且默认在删除 CR 的同时删除备份的文件。若 v1.1.3 及之后版本的用户希望保持该行为，需要设置该字段为 `Delete`。
```

**Pull request merged**